### PR TITLE
liquid templates

### PR DESCRIPTION
**Pull request by coryodaniel**

### CODE REVIEW

File: aws-lambda/massdriver.yaml [hunk 1]
```diff
@@ -130,4 +130,4 @@ ui:
   observability:
     ui:order:
       - retention_days
-      - x-ray
```
Comment (akasprzok) [inline aws-lambda/massdriver.yaml:133]: The hero we need.

File: kubernetes-cronjob/src/chart/Chart.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 apiVersion: v2
-name: <md .Name md>
+name: { { name } }
```
Comment (akasprzok) [inline kubernetes-cronjob/src/chart/Chart.yaml]:
```suggestion
name: {{ name }}
```

File: kubernetes-deployment/src/chart/Chart.yaml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 apiVersion: v2
-name: <md .Name md>
+name: { { name } }
```
Comment (akasprzok) [inline kubernetes-deployment/src/chart/Chart.yaml]:
```suggestion
name: {{ name }}
```

Dunno about go, but the elixir liquid lib will choke on that space.

File: kubernetes-job/src/chart/Chart.yaml [hunk 4]
```diff
@@ -1,5 +1,5 @@
 apiVersion: v2
-name: <md .Name md>
+name: { { name } }
```
Comment (akasprzok) [inline kubernetes-job/src/chart/Chart.yaml]:
```suggestion
name: {{ name }}
```

File: terraform-module/.github/workflows/build.yaml [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+        uses: dsaltares/fetch-gh-release-asset@0.0.8
+        with:
+          repo: "massdriver-cloud/massdriver-cli"
+          file: "^mass-.*-linux-amd64\\.tar\\.gz$"
+          regex: true
+
+      - name: Build
+        env:
+          MASSDRIVER_API_KEY: ${{"{{"}} secrets.MASSDRIVER_API_KEY {{"}}"}}
+        run: tar -xzf mass-*.tar.gz -C /usr/local/bin && rm -rf mass-*.tar.gz .git* && mass bundle build massdriver.yaml
```
Comment (akasprzok) [inline terraform-module/.github/workflows/build.yaml:24]:
```suggestion
      - name: Install Massdriver CLI
        uses: massdriver-cloud/actions/setup@v2.1
      - name: Build bundle
        run: mass bundle build massdriver.yaml
```

Comment (akasprzok) [reply terraform-module/.github/workflows/build.yaml:24]: Not tested, but should be something like that.

File: terraform-module/.github/workflows/publish.yaml [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+        uses: dsaltares/fetch-gh-release-asset@0.0.8
+        with:
+          repo: "massdriver-cloud/massdriver-cli"
+          file: "^mass-.*-linux-amd64\\.tar\\.gz$"
+          regex: true
+
+      - name: Publish
+        env:
+          MASSDRIVER_API_KEY: ${{"{{"}} secrets.MASSDRIVER_API_KEY {{"}}"}}
+        run: tar -xzf mass-*.tar.gz -C /usr/local/bin && rm -rf mass-*.tar.gz .git* && mass bundle build && mass bundle publish
```
Comment (akasprzok) [inline terraform-module/.github/workflows/publish.yaml:25]:
```suggestion
      - name: Install Massdriver CLI
        uses: massdriver-cloud/actions/setup@v2.1
      - name: Publish bundle
        run: mass bundle build && mass bundle publish
```

File: kubernetes-deployment/src/chart/Chart.yaml (hunk 3 above)
Comment (coryodaniel) [reply kubernetes-deployment/src/chart/Chart.yaml]: it will VS code is being an ass.

File: terraform-module/.github/workflows/build.yaml (hunk 5 above)
Comment (coryodaniel) [reply terraform-module/.github/workflows/build.yaml:24]: We can make those changes when we add the github actions to the rest of the templates. This is straight copy/pasta from cli

File: terraform-module/.github/workflows/publish.yaml (hunk 6 above)
Comment (coryodaniel) [reply terraform-module/.github/workflows/publish.yaml:25]: We can make those changes when we add the github actions to the rest of the templates. This is straight copy/pasta from cli

**Pull request merged**